src/App.js

```diff
@@ -1,12 +1,21 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {  BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
 import HomePage from './page/home/HomePage';
+import AboutPage from './page/about/AboutPage';
```

Comment:
```suggestion
import About from './page/about/About';
```
sửa lại như này giúp a, Home, About, Contact thôi nhé, không cần page ở phía sau đâu